src/producer.rs

```diff
@@ -235,7 +257,10 @@ mod tests {
                     },
                     ItemType::Path(ref p) => {
                         elem.1 && p.ends_with(elem.2)
-                    }
+                    },
+                    ItemType::Buffers(_) => {
+                        true
```

Comment:
This should never happen, right? So return `false` here so that if it happens the assertion fails